spec/models/user_spec.rb

```diff
@@ -1,10 +1,18 @@
 require 'rails_helper'
 
-RSpec.describe User, type: :model do
-  it { should have_valid(:first_name).when('Foo') }
-  it { should_not have_valid(:first_name).when(nil, '') }
+describe User do
+  it "should have a first_name attribute" do
+    should have_valid(:first_name).when('Foo')
+  end
+  it "should not have a blank first_name attribute" do
+    should_not have_valid(:first_name).when(nil, '')
+  end
 
-  it { should have_valid(:first_name).when('Bar') }
-  it { should_not have_valid(:first_name).when(nil, '') }
+  it "should have a last_name attribute" do
+    should have_valid(:last_name).when('Bar')
+  end
+  it "should not have a blank last_name attribute" do
+     should_not have_valid(:last_name).when(nil, '')
+   end
```

Comment:
`end` at 16, 3 is not aligned with `it "should not have a blank last_name attribute" do` at 14, 2.